src/System.ServiceModel.Http/tests/ServiceModel/BasicHttpBindingTest.cs

```diff
@@ -169,6 +169,7 @@ public static void Name_Property_Sets(string value)
     [Theory]
     [InlineData(null)]
     [InlineData("")]
+    [ActiveIssue(1449)]
```

Comment:
We could, my thinking was that I can't remember the last time a unit test failed for a product issue, and Ron's PR is going to pave right over this adding the new Wcf conditional fact that will do what you are requesting in a much more elegant way. So we would only lose coverage for a few days at most.